tools/aws_lab_setup/provision_lab.yml

```diff
@@ -1,11 +1,29 @@
+- name: Create users if not defined
+  hosts: localhost
+  connection: local
+  become: no
+  gather_facts: no
+  roles:
+    - student_users
+
 - name: Create lab instances in AWS
   hosts: localhost
   connection: local
   become: no
   gather_facts: no
+  vars:
+    teardown: false
+    create_lab: true
   roles:
     - manage_ec2_instances
```

Comment:
These shouldn't be two separate plays. Please merge.

Reply:
plays? you mean the variables? just use one or the other?

Reply:
I mean that these two plays have identical declarations. You can call both roles in one play rather then have two as you have here. The vars for `manage_ec2_instances` because those should be ignored by any others if you managed your variable naming properly.

Reply:
Example of play calling multiple roles:

https://github.com/ansible/ansible-examples/blob/master/wordpress-nginx/site.yml

Reply:
I think I fixed this but you need to recheck when I submit the commit